DOCS-716 Azure Status and Count Metric (#6947)
DataDog/documentation: content/en/integrations/faq/azure-status-metric.md

---
title: Azure Status and Count Metrics
kind: faq
aliases:
  - /integrations/faq/azure-vm-status-is-not-reporting
---

## Overview

Datadog generates two additional metrics for each resource monitored with the [Azure integration][1]: `azure.*.status` and `azure.*.count`. For example, Azure Virtual Machines monitored by Datadog reports `azure.vm.status` and `azure.vm.count`. These two metrics cover similar information.

The `azure.*.count` metric is an improvement over `azure.*.status`, which will be deprecated.

## .count metric

> The Azure .count metric is **currently in private beta**. Contact [Datadog support][2] to have it enabled for your account.

Reviewer: ```suggestion
The `azure*.count` metric is **currently in private beta**. Contact [Datadog support][2] to have it enabled for your account.
```

The `azure.*.count` metric provides two fundamental pieces of information:

- The number of resources of that type.
- The status of each resource as reported by Azure.

The `azure.*.count` metric is created in the same namespace as the other metrics for that resource type, for example: `azure.network_loadbalancers.count`. It includes all of the same metadata tags as the other metrics in that namespace, plus as additional tag for `status`.

### Use cases

Use the `azure.*.count` metric to:

- Create a view of the number of Virtual Machines broken out by their status over time by graphing `azure.vm.count` over everything and summing by `status`.

> - Create query widgets in dashboards to show the number of running or stopped VMs.

Reviewer 2: We should maybe generalize this one instead of focusing on VMs again.

"Create query widgets in dashboards to display the number of a given resource type. Use any available tags to scope the count to a relevant aggregation- such as region, resource group, kind, or status."

> - Create query widgets in dashboards to show the number of running or stopped VMs.

Author: ```suggestion
- Create query widgets in dashboards to display the number of a given resource type. Use any available tags to scope the count to a relevant aggregation such as region, resource group, kind, or status.
```

- Create monitors to alert you about the status of different Azure resources.

## .status metric

> The Azure .status metric is the previous solution for this same type of information. It reports the number of available resources for each Azure resource type.

Reviewer 3: What does "previous" mean in this context? Was it deprecated? Is it no longer preferred?

Author: No longer the best method but still available.

> The Azure .status metric is the previous solution for this same type of information. It reports the number of available resources for each Azure resource type.

Reviewer: ```suggestion
The `azure.*.status` metric is the previous solution for this same type of information. It reports the number of available resources for each Azure resource type.
```
